docs/CONFIG-PROPERTIES.md

```diff
@@ -6,6 +6,7 @@
 | timer.config.interval | integer in seconds | 60 | how frequently device gets config |
 | timer.metric.interval  | integer in seconds | 60 | how frequently device reports metrics |
 | timer.metric.diskscan.interval  | integer in seconds | 300 | how frequently device should scan the disk for metrics |
+| timer.location.interval | integer in seconds | 20 | how frequently device reports geographic location information (to controller and to local profile server) |
```

Comment:
Once every 20 seconds is too high. How about a default of once every 60 minutes? And a minimum allowed interval of 5 minutes?

Reply:
Basically any interval configured that is lower than 5 minutes shall round up default to 5 minutes.

Reply:
@gkodali-zededa that might mean two timers; one for local (sent to local profile server and available from 169.254.169.254) where 20 seconds is a reasonable default, and one for reporting to the controller where we have to be concerned about load/scale.

Reply:
Yes, if required it makes sense to have two different timers. Fetch at the interval that is minimum of these two.

Reply:
OK, I will split into something like `timer.location.cloud.interval` and `timer.location.app.interval`.

Reply:
Sounds good. Also can you please change the default cloud export interval to 60 minutes and then the smallest cloud export interval to 5 minutes?